src/components/App.js

```diff
@@ -7,31 +7,34 @@ import { connect } from 'react-redux'
 import BetaBanner from './BetaBanner'
 import BetaModal from './BetaModal'
 import Disclaimer from './Disclaimer'
+import Feedback from './Feedback'
 import Footer from './Footer'
 import Glossary from './Glossary'
 import Header from './Header'
-import { hideModal } from '../actions/modal'
 
 const App = ({ appState, children, dispatch, location }) => (
   <div className="site">
     <Disclaimer />
-    <BetaBanner />
+    <BetaBanner dispatch={dispatch} />
```

Comment:
Ya, its definitely better to have `onEvent` handlers passed in when possible. I like that it makes the components more modular and re-usable.